jaas/pom.xml

```diff
@@ -249,7 +249,6 @@
             <groupId>joda-time</groupId>
             <artifactId>joda-time</artifactId>
             <version>2.4</version>
```

Comment:
I think the version can be missed and will be controlled and in sync with Spring Boot version